src/main/java/com/mewna/catnip/Catnip.java

```diff
@@ -116,6 +123,67 @@ static String getShardCountUrl() {
     @Nullable
     User selfUser();
     
+    default void setPresence(@Nonnull final Presence presence) {
```

Comment:
Should probably name this as `Catnip#presence` instead, mainly just to keep in with the rest of the API not using `get`/`set`. I'm open to the idea of leaving it as-is, just it feels a bit weird.